OptimoveSDK/Sources/Classes/EmbeddedMessaging/Models/EmbeddedMessage.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+public struct EmbeddedMessage: Codable {
+    public let customerId: String
+    public let isVisitor: Bool
+    public let templateId: Int64
+    public let title: String
+    public let content: String?
+    public let media: String?
+    public let readAt: Int?
```

Comment:
Are these types aligned across SDKs? (e.g. are we giving dates in some and timestamps in others)

Reply:
Hey @adam-b-optimove @daniel-c-opti what was the outcome RE alignment?